src/dmqproto/client/README.rst

```diff
@@ -7,8 +7,8 @@ This package contains the client to connect to a Distributed Message Queue
 currently supports two protocols, known as the "legacy protocol" and the "neo
 procotol". Detailed documentation about the workings of swarm-based clients and
 the two protocols can be found
-`here (legacy protocol) <https://github.com/sociomantic-tsunami/swarm/blob/v4.x.x/src/swarm/README_client.rst>`_
-and `here (neo protocol) <https://github.com/sociomantic-tsunami/swarm/blob/v4.x.x/src/swarm/README_client_neo.rst>`_.
+`here (legacy protocol) <https://github.com/sociomantic-tsunami/swarm/blob/v5.x.x/src/swarm/README_client.rst>`_
+and `here (neo protocol) <https://github.com/sociomantic-tsunami/swarm/blob/54.x.x/src/swarm/README_client_neo.rst>`_.
```

Comment:
>54.x.x

I don't know what's going on here. I've force pushed to this branch to fix this twice now, to no avail. Possibly I'm mad or possibly github is being wierd :/